.github/workflows/release.yml

```diff
@@ -41,35 +41,39 @@ jobs:
             platform: win32
             arch: x64
             regular_build: 'true'
+            cross: false
           - os: windows-latest
             rust-target: aarch64-pc-windows-msvc
             platform: win32
             arch: arm64
+            cross: false
           - os: ubuntu-20.04
-            rust-target: x86_64-unknown-linux-gnu
+            rust-target: x86_64-unknown-linux-musl
```

Comment:
this can help with rhel. but i heard that some libc function in musl is inefficient. we can observe if there is any slow down after changing this.